src/node-red/database/index.md

```diff
@@ -0,0 +1,19 @@
+---
+eleventyNavigation:
+  key: Database
+meta:
+  title: Node-RED Database Integration Guides
+  description: Explore database integration guides for Node-RED, including PostgreSQL, MongoDB, InfluxDB, DynamoDB, and TimescaleDB
+  keywords: node-red, databases, integration, PostgreSQL, MongoDB, InfluxDB, DynamoDB, TimescaleDB
+---
+
+# Database Integration Guides
+
+Welcome to the Database Integration Guides section. Here, you'll find comprehensive tutorials on integrating various databases with Node-RED. Whether you're working with SQL or NoSQL databases, or need to handle time-series data, these guides will help you connect, store, query, and visualize data effectively.
+
+- [Using PostgreSQL with Node-RED](./postgresql.md)
```

Comment:
I think you can use eleventyNav to dynamically solve this.

Reply:
Done !